plugins/autosuspend.koplugin/main.lua

```diff
@@ -111,7 +111,9 @@ end
 AutoSuspend:init()
 
 local AutoSuspendWidget = WidgetContainer:new{
-    name = "AutoSuspend",
+    name = "autosuspend",
+    fullname = _("Auto suspend"),
+    description = _([[Suspends the device when no interaction for some time.]]),
```

Comment:
Suspends the device after a period of inactivity?

(Merely grammatically speaking, suspends the device when no interaction *has occurred* for some time.)